packages/core/src/App/AppContent.tsx

```diff
@@ -37,16 +37,29 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
                 rudderstackKey: process.env.RUDDERSTACK_KEY,
             };
             Analytics.initialise(config);
+            const ppc =
```

Comment:
I wonder what 'ppc' stands for :) many developers will also wonder. 'pay-per-click'?
Can we name this variable semantically (in a human-readable way) please? 🙏🏻
*same for client-store

Reply:
i like it to be `Pizza Per Client` 🍕

Reply:
resolved by changing name from `ppc` to `ppc_campaign_cookies`

Reply:
great, thanks